students/13th/honghyojeong/js/login.js

```diff
@@ -7,28 +7,28 @@ const loginSubmitBtn = document.getElementById('submit-btn');
 
 
 // 아이디, 패스워드 
-for(let i = 0; i < inputTags.length; i++){
-    inputTags[i].addEventListener('keyup', function(){
-        let loginInputValue = loginInput.value;
-        let passwordInputValue = passwordInput.value;
-        
-        loginInputValue.length > 0 && passwordInputValue.length > 0 ? 
-        loginSubmitBtn.style.backgroundColor = '#68c9e9' : 
-        loginSubmitBtn.style.backgroundColor = 'rgba(0, 149, 246, 0.3)';
-    });
+for (let i = 0; i < inputTags.length; i++) {
+	inputTags[i].addEventListener('keyup', function () {
+		let loginInputValue = loginInput.value;
+		let passwordInputValue = passwordInput.value;
+
+		loginInputValue.length > 0 && passwordInputValue.length > 0 ?
+			loginSubmitBtn.style.backgroundColor = '#68c9e9' :
+			loginSubmitBtn.style.backgroundColor = 'rgba(0, 149, 246, 0.3)';
+	});
 }
 
-loginSubmitBtn.addEventListener('click', function(){
-    let loginInputValue = loginInput.value;
-    let passwordInputValue = passwordInput.value;
+loginSubmitBtn.addEventListener('click', function () {
+	let loginInputValue = loginInput.value;
+	let passwordInputValue = passwordInput.value;
 
-    if(loginInputValue.indexOf('@') === -1){
-        loginInput.style.borderColor = '#ff7777';
-        return alert('이메일 형식을 지켜주세요.')
-    } else loginInput.style.borderColor = '#ddd';
+	if (loginInputValue.indexOf('@') === -1) {
```

Comment:
```suggestion
	if (!loginInputValue.includes('@')) {
```
indexOf 메소드를 사용해서 요소가 있고 없음을 조건으로 쓰실 수 있지만, 이럴 경우에는 .includes 메소드를 사용해보는 것도 좋을 것 같습니다 :) 아래 링크 참고 부탁드립니다 
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/includes